device-code-flow-console/MyInformation.cs

```diff
@@ -46,7 +46,7 @@ public MyInformation(PublicClientApplication app, HttpClient client)
         /// <summary>
         /// Scopes to request access to the protected Web API (here Microsoft Graph)
         /// </summary>
-        private static string[] Scopes { get; set; } = new string[] { "User.Read", "User.ReadBasic.All"};
+        private static string[] Scopes { get; set; } = new string[] { "User.Read", "User.ReadBasic.All"}; // new string[] { "" };
```

Comment:
>// [](start = 106, length = 2)

remove?

Reply:
absolutely. fixed

---
In reply to: [250375425](https://github.com/Azure-Samples/active-directory-dotnetcore-devicecodeflow-v2/pull/6#discussion_r250375425) [](ancestors = 250375425)